cpp/src/Ice/StringUtil.cpp

```diff
@@ -4,7 +4,7 @@
 #include "Ice/StringConverter.h"
 #include <cassert>
 #include <cstring>
-#include <string.h> // NOLINT(modernize-deprecated-headers):for strerror_r
```

Comment:
There should be a space after the colon here.
We do this correctly everywhere else I could find.